tests/test_flows.py

```diff
@@ -266,6 +269,13 @@ def check_topology_notification(self, eth_src: str, neighbors: list,
 
         return True
 
+    def safe_check_obj_attribute(self, obj, attrib_name, expected_val, fail_str):
```

Comment:
*Nitpick*: please document arguments types. Makes usage of this method easier for others.